packages/terra-consumer-nav/translations/nl-BE.json

```diff
@@ -0,0 +1,6 @@
+{
+  "Terra.Consumer.NavHelp.help": "Info",
+  "Terra.Consumer.UserProfile.Modal.title": "Settings",
+  "Terra.Consumer.UserProfile.signout": "Sign Out",
```

Comment:
I believe values here should be in nl-BE. Settings =>  "instellingen" something.

Reply:
Since I haven't got the translations from translation team, the jira is https://jira.cerner.com/browse/TRANS-5085

Reply:
The translations team has finished the translations and those can now be used in this PR.